{
    "type": "bundle",
    "id": "bundle--57fa0c0b-adbd-44f4-ae24-b05a74c1274d",
    "spec_version": "2.0",
    "objects": [
        {
            "object_marking_refs": [
                "marking-definition--fa42a846-8d90-4e51-bc29-71d5b4802168"
            ],
            "id": "relationship--c54e60c1-728d-4211-b309-89f928ae7f0a",
            "type": "relationship",
            "created": "2020-05-18T17:31:39.490Z",
            "created_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5",
            "external_references": [
                {
                    "source_name": "McAfee Maze March 2020",
                    "url": "https://www.mcafee.com/blogs/other-blogs/mcafee-labs/ransomware-maze/",
                    "description": "Mundo, A. (2020, March 26). Ransomware Maze. Retrieved May 18, 2020."
                }
            ],
            "modified": "2020-06-24T01:40:07.572Z",
            "description": "[Maze](https://attack.mitre.org/software/S0449) has gathered all of the running system processes.(Citation: McAfee Maze March 2020)\t",
            "relationship_type": "uses",
            "source_ref": "malware--d9f7383c-95ec-4080-bbce-121c9384457b",
            "target_ref": "attack-pattern--8f4a33ec-8b1f-4b80-a2f6-642b2e479580",
            "x_mitre_version": "1.0",
            "x_mitre_modified_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5"
        }
    ]
}